addons/web/static/src/js/view_list.js

```diff
@@ -2305,13 +2305,9 @@ instance.web.list.Binary = instance.web.list.Column.extend({
         var text = _t("Download");
         var value = row_data[this.id].value;
         var download_url;
-        if (value && value.substr(0, 10).indexOf(' ') == -1) {
-            download_url = "data:application/octet-stream;base64," + value;
```

Comment:
This url was made to avoid downloading again the content. if the goal is to keep the filename, we can use [download attribute](http://stackoverflow.com/a/6943481/147904)